src/h5web/visualizations/nexus/utils.ts

```diff
@@ -105,3 +106,13 @@ export function getDatasetLabel(
 
   return datasetName;
 }
+
+export function normalizeSilxStyle(
+  silxScaleType: 'linear' | 'log' | undefined
+): ScaleType | undefined {
+  if (!silxScaleType) {
+    return undefined;
+  }
+
+  return silxScaleType === 'log' ? ScaleType.Log : ScaleType.Linear;
```

Comment:
I believe we could lowercase the values of the `ScaleType` enum to avoid having to resort to this utility function.

Reply:
We just need to check what SILX uses for `SymLog`, whether it's `symlog` or something else.

Reply:
According to @t20100, we can expect `symlog` as a possible scale type.